practice/src/javascripts/helpers/api.js

```diff
@@ -0,0 +1,27 @@
+import { API } from '../constants/apiUrl';
+
+const APIHelper = {
+  /**
+   * Fetches data from the API using the specified endpoint.
+   * It defines an asynchronous function 'get' to fetch data from the API.
+   * @param {string} - the API endpoint
+   * @returns {Promise<Object>} - the JSON response
+   */
+  async get(endpoint) {
+    try {
+      const response = await fetch(`${API.BASE_URL}/${endpoint}`);
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch products');
+      }
+
+      const data = await response.json();
```

Comment:
why do you need to use await here?

Reply:
`await ` is used here to handle the asynchronous nature of reading and parsing the JSON response from the API.

Reply:
I have some mistakes, so I have fixed it